app/plugins/miscellanea/appearance/index.js

```diff
@@ -273,7 +277,15 @@ volumioAppearance.prototype.createThumbnailPath = function () {
           if(pluginFiles.every(function(f) { return dataFiles.includes(f); })) {
             return null;
           } else {
-            return libQ.nfcall(fs.copy, __dirname + '/backgrounds', backgroundPath);
+            fs.copy(__dirname + '/backgrounds', backgroundPath, function(err) {
```

Comment:
This change does a few things I'm not sure about:
1. It moves the default background selection to before the thumbnail generation (previously it was afterwards, although I'm not sure it matters)
2. It will always run the thumbnail generation, even if the copy operation fails.
3. It does not wait for the result of setting the default background, which is asynchronous (this may have been a pre-existing problem).

Reply:
1) does not matter
2) that should not be an issue
3) that is not a problem as well